network-config-analyzer/PoliciesFinder.py

```diff
@@ -0,0 +1,135 @@
+#
+# Copyright 2020- IBM Inc. All rights reserved
+# SPDX-License-Identifier: Apache2.0
+#
+
+from ruamel.yaml import YAML
+from collections import deque
+from bisect import insort
+from CmdlineRunner import CmdlineRunner
+from NetworkConfig import NetworkConfig, PoliciesContainer
+from NetworkPolicy import NetworkPolicy
+from K8sNetworkPolicy import K8sNetworkPolicy
+from CalicoNetworkPolicy import CalicoNetworkPolicy
+from IstioNetworkPolicy import IstioNetworkPolicy
+from K8sPolicyYamlParser import K8sPolicyYamlParser
+from CalicoPolicyYamlParser import CalicoPolicyYamlParser
+from IstioPolicyYamlParser import IstioPolicyYamlParser
+
+
+class PoliciesFinder:
+    """
+    This class is responsible for finding the network policies in the relevant input resources
+    The class contains several ways to build the set of policies (from cluster, from file-system, from GitHub).
+    """
+    def __init__(self):
+        self.policies_container = PoliciesContainer(policies={}, sorted_policies=[], profiles={}, allowed_labels=set())
+        self._parse_queue = deque()
+        self.type = NetworkConfig.ConfigType.Unknown
+        self.peer_container = None
+
+    def set_peer_container(self, peers):
+        self.peer_container = peers
```

Comment:
```suggestion
    def set_peer_container(self, peer_container):
        self.peer_container = peer_container
```
Also, add some documentation, including param types.